storages/redis-storage/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "gluesql_redis_storage"
+authors = ["Taehoon Moon <taehoon.moon@outlook.com>"]
+version.workspace = true
+edition.workspace = true
+description.workspace = true
+license.workspace = true
+repository.workspace = true
+documentation.workspace = true
+
+[dependencies]
+gluesql-core.workspace = true
+async-trait = "0.1"
+serde = { version = "1", features = ["derive"] }
+redis = "0.23.3"
+serde_json = "1.0.105"
+chrono = { version = "=0.4.31", features = ["serde", "wasmbind"] }
+futures = "0.3"
+toml = "0.8.6"
```

Comment:
`toml` is currently only used in `tests/` so we can declare it in `dev-dependencies` rather than `dependencies`.